src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Machine {
+
+    public static void doWork() {
+
+    }
+
+    public static void stopWork() {
```

Comment:
Similarly, the method 'stopWork' should not be static for the same reasons as 'doWork'. Each subclass should provide its own implementation.